aws_lambda_decorators/validators.py

```diff
@@ -125,34 +128,23 @@ def validate(self, value=None):
             return True
 
         if isinstance(value, (float, int)):
-            return self._minimum <= value
+            return self._condition <= value
 
         return False
 
-    def message(self, value=None):
-        """
-        Gets the formatted error message for a failed minimum value check
-
-        Args:
-            value (int): The validated value
-
-        Returns:
-            The error message
-        """
-        return f"{value} is smaller than minimum value ({self._minimum})"
-
 
-class Maximum:  # noqa: pylint - too-few-public-methods
+class Maximum(Validator):  # noqa: pylint - too-few-public-methods
     """Validation rule to check if a value is less than a maximum value."""
 
-    def __init__(self, maximum: (float, int)):
+    def __init__(self, maximum: (float, int), error_message=None):
         """
         Set the maximum value.
 
         Args:
             maximum (float, int): The maximum value.
+            error_message (str): A custom error message to output if validation fails
         """
-        self._maximum = maximum
+        Validator.__init__(self, error_message or "'{value}' is bigger than maximum value '{condition}'", maximum)
```

Comment:
Same as the previous. Greater than instead of bigger than.

Reply:
made the changes. Could you check again please?